projects/ng-connection-service/package.json

```diff
@@ -27,10 +27,10 @@
   },
   "homepage": "https://github.com/ultrasonicsoft/ng-connection-service#readme",
   "peerDependencies": {
```

Comment:
@ultrasonicsoft I tested it with Angular version 15, and it works. However, since version 9.0.3 is compatible with Angular 9 to 15, I did not test it with Angular versions earlier than 15. Hence, I prepared a version table [see here](https://github.com/ultrasonicsoft/ng-connection-service?tab=readme-ov-file#version-support).